Classes/ViewHelpers/Be/ModuleContainerViewHelper.php

```diff
@@ -45,7 +45,7 @@ public function render($pageTitle = '')
         $doc = $this->getDocInstance();
         $this->pageRenderer->backPath = '';
         $this->pageRenderer->loadExtJS();
-        $this->pageRenderer->addCssFile('sysext/t3skin/extjs/xtheme-t3skin.css');
+        $this->pageRenderer->addCssFile('sysext/core/Resources/Public/ExtJs/xtheme-t3skin.css');
```

Comment:
We'll still have to support TYPO3 7 for a while, so please add condition to support both version similar to, but with correct exact versions where the changed appeared:
 
```php

// From TYPO3 7.4.0 onward we must use EXT prefix
if (version_compare(TYPO3_version, '7.4.0', '>=')) {
    // bla bla
} else {
    // But for TYPO3 6.2 family, we still have to use old style
    // bla bla
}
```

Reply:
Will do soon!